File: app/libs/DSeekBar/src/main/java/com/hdd/seekbar/DSeekFunction.java
package com.hdd.seekbar;

/**
 * Create on 2019-12-15
 *
 * @author duonghd
 */

public interface DSeekFunction {

    DSeekBar setTotalDuration(int totalDuration);

    DSeekBar setDuration(int duration);

    DSeekBar setDSeekListener(DSeekBar.DSeekListener dSeekListener);

    int getTotalDuration();

    int getDuration();
}

File: app/libs/DSeekBar/src/main/java/com/hdd/seekbar/DSeekFunction.kt
package com.hdd.seekbar

/**
 * Create on 2/4/21
 * @author duonghd
 */

interface DSeekFunction {
    fun setTotalDuration(totalDuration: Long): DSeekBar

    fun setDuration(duration: Long): DSeekBar

    fun setDSeekListener(dSeekListener: DSeekListener): DSeekBar

    fun getTotalDuration(): Long

    fun getDuration(): Long
}